styles.css

```diff
@@ -65,3 +73,23 @@ main {
   right: 0px;
   width: 40px;
 }
+
+.navigation {
+  position: fixed;
+  display: flex;
+  justify-content: space-around;
+  width: 100vw;
+  bottom: 0vw;
+  border: 3px solid rgba(9, 8, 8, 0.72);
+  margin-bottom: 0;
+  background-color: rgba(128, 128, 128, 0.973);
+}
+
+.nav {
```

Comment:
maybe give your icons a bit space inside the footer.

Reply:
ok thx :)